database/models/Site.js

```diff
@@ -0,0 +1,39 @@
+const { DataTypes, Model } = require("sequelize")
+
+module.exports = (sequelize) => {
+  class Site extends Model {
+    static associate(models) {
+      const { User, SiteMember } = models
+      Site.belongsToMany(User, {
+        through: SiteMember,
+        onUpdate: "CASCADE",
+        onDelete: "CASCADE",
+      })
```

Comment:
double check if this means that when a site is deleted/updated, all associated users are deleted/updated